sklearn/feature_selection/univariate_selection.py

```diff
@@ -700,6 +754,18 @@ class GenericUnivariateSelect(_BaseFilter):
     pvalues_ : array-like, shape=(n_features,)
         p-values of feature scores, None if `score_func` returned scores only.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import load_breast_cancer
+    >>> from sklearn.feature_selection import GenericUnivariateSelect, chi2
+    >>> X, y = load_breast_cancer(return_X_y=True)
+    >>> X.shape
+    (569, 30)
+    >>> transformer = GenericUnivariateSelect(chi2, 'fpr', param=0.01)
```

Comment:
Maybe use ``percentile`` or ``k_best`` here?  These might be easier for many users to understand. (optional)

Reply:
Done!